File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.java
class Solution {
    public int maxIceCream(int[] costs, int coins) {
        Arrays.sort(costs);
        int count=0;
        for(int i=0;i<costs.length;i++)
        {
            if(costs[i]<=coins)
            {
                count++;
                coins = coins-costs[i];
            }
        }
        return count;
        
    }
}

File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.py
class Solution:
    def maxIceCream(self, costs: List[int], coins: int) -> int:
        costs.sort()
        count =0
        for i in range(len(costs)):
            if costs[i]<=coins:
                count+=1
                coins=coins-costs[i]
        return count
        